### ISSUE TITLE
Latest patch release (6.0.2) introduces breaking change, import error at runtime

### ISSUE THREAD

**Issue by elertan** [3846masa/http-cookie-agent#844]
Hello,

We are using https://github.com/3846masa/axios-cookiejar-support which has a dependency on this package at "^6.0.1".
However the changes introduced in v6.0.2 has a breaking change that causes our application to produce an error at runtime.

```
SyntaxError: Cannot use 'import.meta' outside a module
     at internalCompileFunction (node:internal/vm:128:18)
     at wrapSafe (node:internal/modules/cjs/loader:1279:20)
     at Module._compile (node:internal/modules/cjs/loader:1331:27)
     at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
     at Module.load (node:internal/modules/cjs/loader:1205:32)
     at Module._load (node:internal/modules/cjs/loader:1021:12)
     at Module.require (node:internal/modules/cjs/loader:1230:19)
     at require (node:internal/modules/helpers:179:18)
     at Object.<anonymous> (/docker/api/node_modules/http-cookie-agent/dist/http/create_cookie_agent.js:8:35)
     at Module._compile (node:internal/modules/cjs/loader:1368:14)
```

After overriding the dependency for axios-cookiejar-support to use the exact v6.0.1 of this package, the runtime error disappears.
```json
"overrides": {
    "axios-cookiejar-support": {
        "dependencies": {
            "http-cookie-agent": "6.0.1"
         }
    }
}
```

**Comment by 3846masa** [comment]
Thanks for reporting. Issue was fixed by v6.0.3. Please try it.

**Comment by elertan** [comment]
It's fixed, thank you!
